app/views/api/v2/common_parameters/main.json.rabl

```diff
@@ -2,4 +2,4 @@ object @common_parameter
 
 extends "api/v2/common_parameters/base"
 
-attributes :created_at, :updated_at, :hidden_value
+attributes :created_at, :updated_at, :hidden_value?
```

Comment:
I'd suggest we keep both (I think this was the case at some point), so we have the new field (the boolean ending in `?`) and the old, useless field.